lib/sift/client.rb

```diff
@@ -34,7 +42,13 @@ def initialize(http_response, http_response_code)
     # == Returns:
     #   true on success; false otherwise
     def ok?
-      0 == @api_status.to_i
+      #if there is no content expected, use HTTP code
+      if Response::HTTP_CODES_WITH_NO_BODY.include? @http_status_code
+        204 == @http_status_code
+      # otherwise use API status
```

Comment:
please see `net/http/responses.rb` for the appropriate consts. 0 ought to be `Net::HTTPSuccess` and 204 ought to be `Net::HTTPNoContent`.

http://ruby-doc.org/stdlib-2.1.1/libdoc/net/http/rdoc/Net/HTTPNoContent.html